test/common/json/json_loader_test.cc

```diff
@@ -122,11 +124,6 @@ TEST(JsonLoaderTest, Integer) {
     EXPECT_EQ(std::numeric_limits<int64_t>::max(), json.getInteger("max"));
     EXPECT_EQ(std::numeric_limits<int64_t>::min(), json.getInteger("min"));
   }
-
-  {
```

Comment:
This was testing Max and Min Integer. Unfortunately, rapidjson doesn't throw an exception during Parsing. We should push this validation to the schema. :(